apps/scan/frontend/src/electrical_testing/electrical_testing_screen.tsx

```diff
@@ -0,0 +1,353 @@
+  undefined
+>['taskStatus'];
+
+const SOUND_INTERVAL_SECONDS = 5;
+
+const Row = styled.div<{ gap?: string }>`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: ${({ gap = 0 }) => gap};
+`;
+
+const Column = styled.div<{ gap?: string }>`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  gap: ${({ gap = 0 }) => gap};
+`;
+
+const Small = styled.span`
+  font-size: 0.45rem;
+`;
+
+const SmallButton = styled(Button)`
+  transform: scale(0.5);
+`;
+
+const ExtraSmall = styled.span`
+  font-size: 0.3rem;
+`;
+
+const PlayPauseButton = styled.button`
+  flex-shrink: 0;
+  background-color: #ddd;
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
+  border: none;
+  cursor: pointer;
+  font-size: 1rem;
+  padding: 0;
+
+  svg {
+    scale: 0.8;
+  }
+`;
+
+function CounterButton() {
+  const [count, setCount] = useState(0);
+
+  return (
+    <Button
+      onPress={() => setCount((prev) => prev + 1)}
+      style={{ transform: 'scale(0.5)' }}
+    >
+      Tap Count: {count}
+    </Button>
+  );
+}
+
+function TaskStatusButton({
+  taskStatus,
+  onPress,
+}: {
+  taskStatus: TaskStatus;
+  onPress: () => void;
+}) {
+  return (
+    <PlayPauseButton onClick={onPress}>
+      {taskStatus === 'running' ? <Icons.Pause /> : <Icons.Play />}
+    </PlayPauseButton>
+  );
+}
+
+function formatTimestamp(timestamp: DateTime): string {
+  return timestamp.toLocal().toFormat('h:mm:ss a MM/dd/yyyy');
+}
+
+function StatusCard({
+  title,
+  statusMessage,
+  body,
+  updatedAt,
+  taskStatus,
+  onToggleRunning,
+}: {
+  title: React.ReactNode;
+  statusMessage?: React.ReactNode;
+  body?: React.ReactNode;
+  updatedAt?: DateTime;
+  taskStatus?: TaskStatus;
+  onToggleRunning?: () => void;
+}) {
+  return (
+    <Card
+      style={{
+        width: '600px',
+        height: '235px',
+        overflow: 'hidden',
+      }}
+    >
+      <Row style={{ height: '100%', alignItems: 'stretch' }}>
+        <Column style={{ flexGrow: 1 }}>
+          <H6 style={{ flexGrow: 0 }}>{title}</H6>
+          {statusMessage && (
+            <Caption
+              style={{
+                flexGrow: 1,
+                overflowWrap: 'anywhere',
+                maxHeight: '2rem',
+                overflow: 'hidden',
+              }}
+            >
+              <Small>{statusMessage}</Small>
+            </Caption>
+          )}
+          {body && <Caption style={{ flexGrow: 1 }}>{body}</Caption>}
+          {updatedAt && (
+            <Caption style={{ flexGrow: 0 }}>
+              <ExtraSmall>{formatTimestamp(updatedAt)}</ExtraSmall>
+            </Caption>
+          )}
+        </Column>
+        {taskStatus && onToggleRunning && (
+          <Column style={{ flexGrow: 0, alignContent: 'center' }}>
+            <TaskStatusButton
+              taskStatus={taskStatus}
+              onPress={onToggleRunning}
+            />
+          </Column>
+        )}
+      </Row>
+    </Card>
+  );
+}
```

Comment:
Not necessary right now but just noting that the VxMark electrical testing app may also get the same updates at which point we can move some of these components to a shared location